WBooks/Screens/BaseViewController.swift

```diff
@@ -0,0 +1,49 @@
+//
+//  BaseViewController.swift
+//  WBooks
+//
+//  Created by Nacho 2 on 22/01/2020.
+//  Copyright © 2020 Wolox. All rights reserved.
+//
+
+import UIKit
+
+class BaseViewController: UIViewController {
+
+    override var preferredStatusBarStyle: UIStatusBarStyle {
+        .lightContent
+    }
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        setNavigationBar()
+        UINavigationBar.appearance().titleTextAttributes = [NSAttributedStringKey.foregroundColor:UIColor.white]
+    }
+    
+    func setTitle(headerTitle: String? = nil) {
+        title = headerTitle
+    }
+    
+    func setLeftButtonImage(imageName: String) {
+        setNavigationLeftButtons([UIBarButtonItem(image: UIImage(named: imageName), style: UIBarButtonItemStyle.plain, target: nil, action: nil)])
+    }
+    
+    func setRightButtonImage(imageName: String) {
+        setNavigationRightButtons([UIBarButtonItem(image: UIImage(named: imageName), style: UIBarButtonItemStyle.plain, target: nil, action: nil)])
+    }
+    
+    func setNavigationBar() {
```

Comment:
You are not setting all navigationBar, just the image, so maybe `setNavigationBarImage` would be better